jpype/imports.py

```diff
@@ -198,7 +198,7 @@ def find_spec(self, name, path, target=None):
             try:
                 # Use forname because it give better diagnostics
                 cls = _jpype.JClass("java.lang.Class").forName(name)
-                return _jpype.JClass(cls)
+                raise ImportError("Class `%s` was found but was not expected" % name)
```

Comment:
This message will always be overwritten by the one of ImportError in line 204, is this intended?

Reply:
Yes intent was to always issue the import error with some cause.  Thus the line above should always have failed.  The question being what to do if it did not fail.  So i issued a new exception saying this should have failed which will be the cause of the actual failure mesg.  I can try to make it cleaner but it should never happen.